### ISSUE TITLE
newObjectEndpoint, and default property initialization (organization, etc) need to work on Concepts / Schemes.

### ISSUE THREAD

**Issue by Lomilar** [cassproject/cass-editor#529]
Postmessage that sets the default properties for new objects needs to propagate to concepts and schemes as well.

**Comment by woodkri** [comment]
The newObjectEndpoint is working on ctitest.

For the template set by postMessage from the parent, right now the CaSS iframe is receiving competency and framework fields. Will concept scheme and concept be added to the postMessage or does CaSS need to translate between framework terms and concept scheme terms here? If the latter, what would be the concept scheme equivalent to ceasn:repositoryDate?

**Comment by Lomilar** [comment]
EcConcept and EcConceptScheme have templates now as well that are used during object creation.

The postMessage just needs to check for concept mode and set fields in the scheme and concept templates in the same fashion.

**Comment by woodkri** [comment]
Right, I see where to set the templates. Right now though, the postMessage from the parent is giving me things like data.framework["schema:publisher"] which isn't the right property for concept schemes. We would either need to translate schema:publisher to dcterms:publisher (and other fields), or the postMessage would need to send us the fields for concept schemes and concepts.

**Comment by Lomilar** [comment]
Should be available on CTITest. See https://github.com/cassproject/cass-editor#provide-templates-for-frameworks-and-competencies for modifications.

**Comment by Lomilar** [comment]
The postMessage should send us the fields for concept schemes and concepts according to the CaSS profile of SKOS. @cwd-mparsons This OK?

**Comment by cwd-mparsons** [comment]
I had copied this code from the competency page, and clearly didn't check the format. 
I updated the postMessage to use dcterms:creator and ceterms:publisher. Changed framework to conceptScheme, and competency to concept. This has been updated in ctitest (and verified with a quick add)

![image](https://user-images.githubusercontent.com/26332112/54842301-941dc200-4c9f-11e9-9286-c773affedcd2.png)

Is something different needed for ceasn:repositoryDate? I don't see thelatter is the SKOS (JSON-LD). Or is it not needed?
